### DISCUSSION TITLE
Y-axis range

### DISCUSSION THREAD

**Discussion by jphilipdev** [klinecharts/KLineChart#148] [Q&A]
Hi, is it possible to control the y-axis range?

In the samples the y-axis changes automatically based on min/max visible values - I would like to manually set the range like on TradingView so it is not "auto":

![image](https://user-images.githubusercontent.com/8587661/126030886-5f63a2dc-7476-4483-a02e-5bb828279b7a.png)

Thanks and the library is awesome!

**Comment by liihuu** [comment]
Future versions will support.

**Reply by jphilipdev** [reply]
Thanks!

**Reply by dooee** [reply]
I'm waiting for the update on this feature. Any info?

**Reply by liihuu** [reply]
I will as soon as possible.

**Reply by emha** [reply]
Do you have any information/update on this? Thanks in advance :)

**Reply by mercior** [reply]
I would also love to see this feature

**Reply by liihuu** [reply]
I'm busy and don't have much spare time to work on this feature, but I'll do.

**Comment by jinusean** [comment]
I'm willing to work on this if you can provide a little guidance, thanks for the awesome project @liihuu

**Reply by dooee** [reply]
Sounds great!! A-Yo!!!

**Comment by liihuu** [comment]
Try the latest version.😄

**Reply by dooee** [reply]
Really awesome. Great job!
It works as I expected in desktop view.
But something doesn't seem to work on mobile. If I hold down the Y-axis region and drag to up/down, it doesn't work as adjusting ratio. It just moves the screen.

**Reply by liihuu** [reply]
mobile not support

**Reply by byeugenius** [reply]
How to change default Y-axis range in my charts?

**Comment by KotlyaroFF** [comment]
How can I add the ability to change the range using the mouse scroll?
